testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -227,6 +329,100 @@ def test_initial_push_replication(setup_client_syncgateway_test, continuous):
 @pytest.mark.listener
 @pytest.mark.syncgateway
 @pytest.mark.replication
+@pytest.mark.session
+@pytest.mark.parametrize("num_docs, need_attachments, replication_after_backgroundApp", [
+    (1000, True, False),
+    (10000, False, False),
+    (10000, False, False)
+])
+def test_push_replication_with_backgroundApp(setup_client_syncgateway_test, num_docs, need_attachments, replication_after_backgroundApp):
+    """
+    1. Prepare LiteServ to have 10000 documents.
+    2. Start continous push replication to push the docs into a sync_gateway database.
+    3. While docs are getting replcate , push the app to the background
+    4. Verify if all of the docs get pushed and replication continous when app goes background
+    """
+
+    sg_db = "db"
+    ls_db = "ls_db"
+    seth_channels = ["ABC", "NBC"]
+
+    cluster_config = setup_client_syncgateway_test["cluster_config"]
+    sg_mode = setup_client_syncgateway_test["sg_mode"]
+    ls_url = setup_client_syncgateway_test["ls_url"]
+    sg_one_admin = setup_client_syncgateway_test["sg_admin_url"]
+    sg_one_public = setup_client_syncgateway_test["sg_url"]
+    liteserv = setup_client_syncgateway_test["liteserv"]
+    liteserv_platform = setup_client_syncgateway_test["liteserv_platform"]
+    device_enabled = setup_client_syncgateway_test["device_enabled"]
+
+    sg_config = sync_gateway_config_path_for_mode("listener_tests/listener_tests", sg_mode)
+    c = cluster.Cluster(config=cluster_config)
+    c.reset(sg_config_path=sg_config)
+
+    # No command to push the app to background on device, so avoid test to run on ios device
+    if((liteserv_platform.lower != "ios" and liteserv_platform.lower != "android") or (liteserv_platform.lower == "ios" and device_enabled)):
+        pytest.skip('This test only valid for mobile')
```

Comment:
This should probably say device

Reply:
This is not answered

Reply:
mmm... basicaly it is valid for device and valid for mobile, just that no command for iOS device to make app background. Changed to message with additional message that it cannot run on IoS device